ads/pubmine.js

```diff
@@ -25,11 +26,25 @@ const pubmineOptional = ['section', 'pt', 'ht'],
  * @param {!Window} global
  */
 function initMasterFrame(data, global) {
+  /*
+   * INSUFFICIENT and UNKNOWN should be treated as INSUFFICIENT
+   * unless state is UNKNOWN and `data-npa-on-unknown-consent=false`
+   */
+  const paUnknown =
+    data['npaOnUnknownConsent'] !== undefined && !data['npaOnUnknownConsent'];
```

Comment:
I've updated the check to explicitly check for a `false` value. 

The mechanics of `data-npa-on-unknown-consent` are not well documented so I falsely assumed it would always be `undefined`|`true`|`false`. A bit more testing shows that it merely pipes whatever string the data value contains (including an empty string if no value is passed).

Reply:
`data-npa-on-unknown-consent` is an attributed supported by your ad vendor. Please make sure that you clearly documented the behavior to your client.

Reply:
Yes I believe our consent states are clearly stated in `pubmine.md` **Consent Support** section.

Reply:
Sounds good. You may also want to update the `Optional Parameters` section of the doc?

Reply:
The amp core attribute `data-npa-on-unknown-consent` seems to automatically add the `npaOnUnknownConsent` parameter--is this something I should treat as a regular optional parameter.

Reply:
Maybe it's not supposed to be a general purpose feature yet? I'm basing most of the functionality as described in this reference: https://support.google.com/adsense/answer/7670312?hl=en

Reply:
That's not an attribute supported by amp core. Instead it's an attribute supported on your end. That's why you're introducing the check on `npaOnUnknownConsent`. 

FYI: AMP converts `data-abc-def` to `data['abcDef']`.

Reply:
Ah, one of the `amp.dev/documentation/` pages led me to that adsense support page, which led me to believe it was a core feature. At any rate, I have it implemented so I'll go ahead and add it.

Reply:
Sounds good. Thanks

Reply:
Added the note. BTW, it was this page that led me to that adsense answer: https://amp.dev/documentation/examples/user-consent/advanced_user_consent_flow/?format=websites -- about half a page up from **Need further explanation?**

It might be good to clarify that `data-npa-on-unknown-consent` is a Adsense-specific implementation and not a poorly documented amp core feature.

Reply:
I looked at the doc, it says ad networks can implement their own behavior, and indicate that the it's for Doubleclick. 

"e.g. default to non-personalized ads without consent, as seen documented here for Doubleclick"

What's your suggestion on the wording here?

Reply:
I guess it wasn't clear to me which part of it was the self-implemented behavior. Perhaps something like "e.g. default to non-personalized ads without consent, as demonstrated with the custom `data-npa-on-unknown-consent` attribute documented here..."